recipes-core/packagegroups/packagegroup-ni-desirable.bb

```diff
@@ -35,6 +35,7 @@ RDEPENDS_${PN} += "\
 	ldd \
 	ltrace \
 	ntpdate \
+	onboard \
```

Comment:
Should we add this to recipes-core/packagegroups/packagegroup-ni-coreimagerepo.bb instead? I think we want to make sure this is available (but not installed by default).

Reply:
I believe that the coreimagerepo package group no longer exists. I think it was sort of replaced by packagefeed-ni-core, and as I understand, we do not want individual packages in there. I couldn't find a better location for this package in the current package group architecture.
Besides, packages in the desirable packagegroup should be available, and not installed by default. These packages are configured to break our CI build if they do not build.

Reply:
OK, my information is probably outdated. I was commenting based on a similar commit @amstewart made for nohz kernel (but I guess that's a package group): https://github.com/ni/meta-nilrt/pull/299/commits/ba66f55a5340aa264e90db8e07738d6169776687

Reply:
I think @gratian is objecting to the `onboard` package being in the extra/ feed, and I agree that it doesn't belong there.

I think it makes sense that we would have a packgegroup for "the XFCE desktop environment" and another for "all graphical packages". So what about doing...

```
packagefeed-ni-core
\-> packagegroup-ni-graphical
    |-> packagegroup-ni-xfce
    |   \-> xfce & deps
    \-> onboard
```

And then the images recipes can just be:
```
nilrt-runmode-rootfs.bb
\-> includes/nilrt-xfce.bb
    \-> IMAGE_INSTALL += "packagegroup-ni-xfce"
```

So everything in `-graphical` (inlcuding XFCE) gets built into the core/ feed, but only the `-xfce` packages get installed to the rootfs image.

Reply:
Addressed in v2. Let me know if that's what you imagined.